plugins/woocommerce/src/Internal/Admin/ReviewsListTable.php

```diff
@@ -42,7 +42,15 @@ class ReviewsListTable extends WP_List_Table {
 	 * @param array|string $args Array or string of arguments.
 	 */
 	public function __construct( $args = [] ) {
-		parent::__construct( $args );
+		parent::__construct(
+			wp_parse_args(
+				$args,
+				[
+					'plural'   => 'product-reviews',
+					'singular' => 'product-review',
```

Comment:
In case you're [comparing with PRP](https://github.com/gdcorp-partners/woocommerce-product-reviews-pro/blob/main/src/admin/class-wc-reviews-list-table.php#L68-L69), these are intentionally slugs and not translateable. I do not believe they should be translated. If you look at the base `WP_List_Table` class where these are used, they are used when creating nonce actions. The `'bulk-product-reviews'` nonce action (used on line 737 of this class) is generated by WordPress core using `wp_nonce_field( 'bulk-' . $this->_args['plural'] )` . If that were translated it would mess up the nonce action.